src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectModal/components/ReceiveTokensItem.tsx

```diff
@@ -31,7 +39,18 @@ export const ReceiveTokensItem = ({ className }: { className?: string }) => {
             <Trans>{badgeTitle} Token</Trans>
           </CartItemBadge>
         </div>
-        <div>{receivedTickets}</div>
+        {buybackDelegateEnabled ? (
+          <div>
+            ≥ {receivedTickets}
+            <TooltipIcon
+              tip={
+                'This project may purchase tokens from a secondary market instead of minting new tokens, which may result in more tokens distributed depending on the swap price.'
```

Comment:
```suggestion
                'Your payment may purchase tokens from a secondary market instead of minting new tokens. You might receive more tokens depending on the swap price.'
```